Add Minecraft+Wikia Commands

fkndean opened the pull request:
Recreated PR for Minecraft Commands.

appelemac commented on src/NadekoBot/Modules/Searches/Searches.cs:
```diff
@@ -464,6 +465,90 @@ await channel.SendMessageAsync($"https://google.com/search?q={ WebUtility.UrlEnc
+                    var sb = new StringBuilder();
+                    sb.AppendLine($"`Server:` {arg.ToString()} 〘Status: {items["status"]}〙");
+                    sb.AppendLine($"`Player List:`");
+                    for (int i = 0; i < items["Playerlist"].Count(); i++)
```
you can should change this to `foreach (var item in items["Playerlist"].Take(5))` and keep a seperate `i`
Mostly for consistency, plus you're calling Count() every time you go through the loop now.

fkndean commented:
thanks doing it now

---- On Sun, 06 Nov 2016 13:30:33 -0800 notifications@github.com wrote ----

@appelemac commented on this pull request.

In src/NadekoBot/Modules/Searches/Searches.cs:

> +                return;
+            }
+            await umsg.Channel.TriggerTypingAsync().ConfigureAwait(false);
+            using (var http = new HttpClient())
+            {
+                http.DefaultRequestHeaders.Clear();
+                try
+                {
+                    string ip = arg.Split(':')[0];
+                    string port = arg.Split(':')[1];
+                    var res = await http.GetStringAsync($"https://api.minetools.eu/query/{Uri.EscapeUriString(ip)}/{Uri.EscapeUriString(port)}").ConfigureAwait(false);
+                    var items = JObject.Parse(res);
+                    var sb = new StringBuilder();
+                    sb.AppendLine($"`Server:` {arg.ToString()} 〘Status: {items["status"]}〙");
+                    sb.AppendLine($"`Player List:`");
+                    for (int i = 0; i < items["Playerlist"].Count(); i++)
you can should change this to foreach (var item in items["Playerlist"].Take(5)) and keep a seperate i
Mostly for consistency, plus you're calling Count() every time you go through the loop now.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.

Kwoth commented:
Can you prepare this for merge, I'll merge it myself when I wake up if not.
